src/math/Sphere.js

```diff
@@ -104,6 +104,20 @@ THREE.Sphere.prototype = {
 
 	},
 
+	intersectsPlane: function ( plane ) {
+
+		// We use the following equation to compute the signed distance from
+		// the center of the sphere to the plane.
+		//
+		// distance = q * n - d
+		//
+		// If this distance is greater than the radius of the sphere,
+		// then there is no intersection.
+
+		return ( ( this.center.dot( plane.normal ) - plane.constant ) <= this.radius );
+
```

Comment:
Shouldn't you be taking the absolute value of the signed distance?

Do you mind adding unit tests to demonstrate the correctness of the two new methods?